src/measure_theory/Meas.lean

```diff
@@ -24,6 +27,34 @@ def of (X : Type u) [measurable_space X] : Meas := ⟨X⟩
 -- -- If `measurable` were a class, we would summon instances:
 -- local attribute [class] measurable
 -- instance {X Y : Meas} (f : X ⟶ Y) : measurable (f : X → Y) := f.2
+
+def Measure : Meas ⥤ Meas :=
+⟨ λX, ⟨@measure_theory.measure X.1 X.2⟩,
+  λX Y f, ⟨measure.map f, measure.measurable_map f f.2⟩,
+  assume ⟨α, I⟩, subtype.eq $ funext $ assume μ, @measure.map_id α I μ,
+  assume X Y Z ⟨f, hf⟩ ⟨g, hg⟩, subtype.eq $ funext $ assume μ, (measure.map_map hg hf).symm ⟩
+
+instance : category_theory.monad Measure.{u} :=
+⟨ ⟨ λX, ⟨@measure.dirac X.1 X.2, measure.measurable_dirac⟩,
```

Comment:
Same here. I don't know by heart which data and axioms occur in which order... now I need to invoke my internal type checker to find out.